csrc/core.cpp

```diff
@@ -111,6 +126,9 @@ void TorchMemorySaver::resume(const std::string& tag) {
             // maybe we can free host memory if needed (currently keep it there to reduce re-alloc time)
         }
 
+        metadata.state = AllocationState::ACTIVE;
+        metadata.allocHandle = newAllocHandle;
```

Comment:
if this line is moved to here, then our " (old)metadata.allocHandle=" log should be changed, o/w we always print new handle

Reply:
Good catch! Fixed it!